tests/validate_yaml_load.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -ex
+
+# Basic yaml syntax check
+yaml_files=$(find -iregex '.*\.\(yaml\|yml\)$')
+for yaml_file in $yaml_files; do
+    echo "$yaml_file"
+    /usr/bin/env python3 -c 'import yaml,sys;yaml.safe_load(sys.stdin)' < $yaml_file
```

Comment:
Is this meant to pass when/if there are no .yaml files in the repo? If so, I am +1 to it.